CHANGELOG.md

```diff
@@ -1,152 +1,86 @@
-
-# Changelog version 2.8.0
+Thanks to Raenore for the help on some of these features, and keyboardturner for the new silhouette logo.
 
 ## Added
-
-- Added a small indicator at the bottom of the tooltip while receiving updates to a profile.
+- Added a new Relations menu located in the Customization section, alongside the moved Automation settings.
+  - This menu lets you edit the preset relations you can have with other players, as well as create new ones.
+  - Those relations are still only personal, and not shared with other players.
+  - The relation will now be shown as a line on the tooltip in addition to the border color. This can be changed to show the relation description, or disabled in Tooltip settings.
+- Added a "Currently" toolbar button letting you open a frame to quickly modify your Currently info.
+- Added a toolbar button to open/close the main Total RP 3 window.
+- Added a target frame button for your own companion to auto-fill the speech window with its name in brackets.
+  - This will apply RP name customization to its speech lines.
+- Added a dropdown to set whether you are open to other players walking up to your character.
+  - This will show as an icon on your tooltip and in your profile characteristics.
+  - Existing tooltip bottom icons have also been changed for clarity.
+    - Book: Profile has an unread description.
+    - Note: You have notes on this profile.
+    - Three people: Walkup friendly.
+- Added a tooltip setting to hide the addon version text and trial indicator separately from the bottom icons.
+- Added the ability to drag and drop personality traits to reorder them
+  - Additional info now uses the same button style rather than drag and dropping the icon itself.
+- Added the ability to duplicate a personality trait or additional info line, as well as convert preset lines into custom.
+  - These are located within the new Options button at the end of the line, alongside deletion of the line.
+  - Preset additional info lines appearing on the tooltip will no longer appear if converted to custom.
+- Added indicators on additional info presets which will appear on tooltips by default.
+- Added the ability to right-click an icon button to copy/paste icons between buttons or copy the name of the selected icon.
+- Added a new background browser, as well as 33 new background options for the "About" section (and Extended documents).
+- The toolbar will now be skinned while using ElvUI.
+- Added X icons, X images and X musics from patch 11.0.
 
 ## Changed
-
-- Chat links will no longer display the added number at the end of the name when sent. (We've also learned that after 1 comes 2, not 12.)
+- The UI has received an extensive makeover, including smoother scrolling in profiles list and directory.
+- The toolbar and target frame buttons have had their icons updated to make their actions clearer.
+  - **TODO: INSERT IMAGE**
```

Comment:
I guess images are going in the visual guide and as such this todo can be removed?